src/org/camunda/latera/bss/connectors/PlanadoV2.groovy

```diff
@@ -203,10 +203,11 @@ class PlanadoV2 {
     }
 
     LinkedHashMap payload = [
-      template_id  : toIntSafe(data.templateId),
-      client_id    : data.clientId,
-      scheduled_at : data.startDate,
-      description  : data.description ?: ''
+      template_id   : toIntSafe(data.templateId),
+      client_id     : data.clientId,
+      scheduled_at  : data.startDate,
+      description   : data.description  ?: '',
+      custom_fields : data.customFields ?: []
```

Comment:
This requires to pass custom fields as list:
```groovy
Map newJob = planadl.createJob(
  templateId: ...,
  clientId: ...,
  customFields: [[
    name: 'someField',
    value: 'Some value'
]])
```

But it's really more convenient better to pass custom fields as Map:
```groovy
Map newJob = planadl.createJob(
  templateId: ...,
  clientId: ...,
  customFields: [
    someField: 'Some value'
])
```

It's how current integration with [OTRS](https://github.com/latera/camunda-ext/blob/master/src/org/camunda/latera/bss/connectors/otrs/types/Ticket.groovy#L137) or [Hydra's REST API](https://github.com/latera/camunda-ext/blob/master/src/org/camunda/latera/bss/connectors/hoper/hydra/Customer.groovy#L58) working

Reply:
We have list of hashes in Planado REST, why don't we pass it? (Option 1)

Reply:
@dolfinus, if option 1 works properly then we use it

Reply:
[key: value] as a way to pass custom fields is used all classes in this project. It's quite strange to see in one of them another approach.

Current solution is not really wrong or not working, but I think it's better to have some similarities between connectors for faster learning.

Reply:
okay, maybe we do it next time

Reply:
Why next time?

Reply:
thanks a lot for your help with our codebase, let it go now